atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweeperServiceImpl.java

```diff
@@ -126,16 +126,18 @@ private SweepBatchConfig buildConfigWithOverrides(
     }
 
     private TableReference getTableRef(String tableName) {
-        WebPreconditions.checkArgument(
-                TableReference.isFullyQualifiedName(tableName), "Table name {} is not fully qualified", tableName);
+        Preconditions.checkArgument(
+                TableReference.isFullyQualifiedName(tableName),
+                "Table name is not fully qualified",
+                UnsafeArg.of("tableName", tableName));
```

Comment:
nit: you can use `LoggingArgs.safeInternalTableName` here.

Reply:
updated both